client/src/components/layout/UserProfile.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react";
+import Fetcher from "./services/Fetcher";
+import { Link } from "react-router-dom";
+
+import MyReviewTile from "./MyReviewTile";
+import DogTile from "./DogTile";
+
+const UserProfile = (props) => {
+  const userId = props.user.id;
+
+  const [user, setUser] = useState({
+    name: "",
+    email: "",
+    dogs: [],
+    reviews: [],
+  });
+
+  const getUserData = async () => {
+    const response = await Fetcher.get("/api/v1/users/");
```

Comment:
To follow proper RESTful HTTP requests we could consider making this request GET api/v1/users/userId since React knows who the signed in user is, that way we are making a proper GET request for one resource of information, rather than api/v1/users which would indicate getting the index of ALL users (which we might want that path available for a different feature)

The app URL can still stay the same as `user-profile`, but the fetch request here could be refactored